packages/react-examples/package.json

```diff
@@ -36,8 +36,7 @@
     "glob": "^7.1.2",
     "jest-snapshot": "~24.9.0",
     "react-app-polyfill": "~1.0.1",
-    "react-test-renderer": "^16.3.0",
-    "storybook-addon-performance": "^0.9.0"
```

Comment:
_Is not removed, moved to a root `package.json`_

Reply:
yeah sorry about the quite big PR... I  know it's quite hard to go through all the changes. appreciate the endurance though 👍